<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>891</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PCCA</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: white;
	border-image: url(:/imagenes/assets/Image.png) 0 0 0 0 stretch stretch;
	background-repeat: no-repeat;
	background-position: center;
}

QGridLayout {
	background-color: rgba(0,0,0,0);
}

.QScrollArea {
	background-color: rgba(0,0,0,0);
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_titulo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Planes de estudio (2 -2022)</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_actualizar_periodo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">.QPushButton {
	color: white;
	background: #FF7A00;
	border-radius: 5px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
         </property>
         <property name="text">
          <string>Actualizar período actual</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>600</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">.QWidget {
	background-color: rgba(234,32,23,0);
}

.QPushButton {
	background: #E6E6E6;
	border-radius: 15px;
}

.QPushButton:hover {
	background: #083C87;
	color: white;
}

.QPushButton:pressed {
	background: #FF7A00;
	color: white;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1007</width>
             <height>602</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <property name="leftMargin">
               <number>30</number>
              </property>
              <property name="rightMargin">
               <number>30</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>30</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="rightMargin">
           <number>60</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_estimar">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton {
	color: white;
	background: #FF7A00;
	border-radius: 25px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
            </property>
            <property name="text">
             <string>Estimar planes de estudio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_cargar">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton {
	color: white;
	background: #FF7A00;
	border-radius: 25px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
            </property>
            <property name="text">
             <string>Ingresar planilla de planificación</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_registrar">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton {
	color: white;
	background: #FF7A00;
	border-radius: 25px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
            </property>
            <property name="text">
             <string>Registrar Nuevo Plan de estudio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_registrar_2">
            <property name="minimumSize">
             <size>
              <width>260</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">.QPushButton {
	color: white;
	background: #FF7A00;
	border-radius: 25px;
}

.QPushButton:hover {
	background: #083C87;
}

.QPushButton:pressed {
	background: #E6E6E6;
}</string>
            </property>
            <property name="text">
             <string>Gestión de estadísticas curriculares</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>assets/logo-diinf.PNG</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>button_estimar</tabstop>
  <tabstop>button_cargar</tabstop>
  <tabstop>button_registrar</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
